server.js

```diff
@@ -27,6 +15,23 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/nominations', (req, res) => {
```

Comment:
I get a server error when I try to load this endpoint :( https://project-express-api-clara.herokuapp.com/nominations